timeboost-sequencer/src/decrypt.rs

```diff
@@ -340,6 +355,10 @@ impl Worker {
 
     /// logic to process a `WorkerRequest::Decrypt` request from the decrypter
     async fn on_decrypt_request(&mut self, round: RoundNumber, incl: InclusionList) -> Result<()> {
+        // reject requests too far into the future without proper round evidence
+        if !incl.evidence().is_valid(round, &self.committee) {
+            return Err(DecryptError::MissingRoundEvidence(round));
+        }
```

Comment:
```suggestion
```
This is redundant. The local evidence has already been checked and by is at most one round away. Evidence validation needs to happen when receiving shared from remote.

Reply:
sounds good, removed.